src/Components/EmailNotificationForm/EmailNotificationForm.js

```diff
@@ -0,0 +1,271 @@
+      { value: 'mo', label: 'Missouri' },
+      { value: 'mt', label: 'Montana' },
+      { value: 'ne', label: 'Nebraska' },
+      { value: 'nv', label: 'Nevada' },
+      { value: 'nh', label: 'New Hampshire' },
+      { value: 'nj', label: 'New Jersey' },
+      { value: 'nm', label: 'New Mexico' },
+      { value: 'ny', label: 'New York' },
+      { value: 'nc', label: 'North Carolina' },
+      { value: 'nd', label: 'North Dakota' },
+      { value: 'oh', label: 'Ohio' },
+      { value: 'ok', label: 'Oklahoma' },
+      { value: 'or', label: 'Oregon' },
+      { value: 'pa', label: 'Pennsylvania' },
+      { value: 'ri', label: 'Rhode Island' },
+      { value: 'sc', label: 'South Carolina' },
+      { value: 'sd', label: 'South Dakota' },
+      { value: 'tn', label: 'Tennessee' },
+      { value: 'tx', label: 'Texas' },
+      { value: 'ut', label: 'Utah' },
+      { value: 'vt', label: 'Vermont' },
+      { value: 'va', label: 'Virginia' },
+      { value: 'wa', label: 'Washington' },
+      { value: 'wv', label: 'West Virginia' },
+      { value: 'wi', label: 'Wisconsin' },
+      { value: 'wy', label: 'Wyoming' }
+    ];
+
+    const options = [
+      <option className="state-name-option" name="state" value="Select" key="Select" disabled={true}>
+        Select
+      </option>
+    ];
+  
+    const stateOptions = states.map(state => {
+      return (
+        <option className="state-name-option" name="state" value={state.value} key={state.label}>
+          {state.label}
+        </option>
+      );
+    });
+
+    return options.concat(stateOptions);
+  }
+
+  render = () => {
+    return (
+      <form className="form-container">
+        <div className="form-header-container">
+          <p className="form-header">State Election Reminders</p>
+        </div>
+        <div className="form-content-container">
+          <div className="label-input-container">
+            <label className="label">
+              First Name<em>*</em>
+              <input type="text" 
+                name="first_name" 
+                value={this.state.first_name}
+                id="first_name" 
+                required="required" 
+                aria-required="true"
+                className="input"
+                onChange={(event) => this.handleInputChange(event)}/>
+            </label>
+          </div>
+          <div className="label-input-container">
+            <label className="label">
+              Last Name<em>*</em>
+              <input type="text" 
+                name="last_name" 
+                value={this.state.last_name}
+                id="last_name" 
+                required="required" 
+                aria-required="true" 
+                className="input"
+                onChange={(event) => this.handleInputChange(event)}/>
+            </label>
+          </div>
+          <div className="label-input-container">
+            <label className="label">State<em>*</em></label>
```

Comment:
It looks like you wrapped only the text here.

Reply:
I've refactored this so that all of the labels are on their own and have a htmlFor attribute.